gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -178,12 +178,8 @@ def create_postgres_root_user
     end
 
     def create_postgres_database
-      run_as_postgres("createdb -E utf8 -O #{username} #{database}")
-    end
-
-    # some overlap with PostgresAdmin
-    def run_as_postgres(cmd)
-      AwesomeSpawn.run("su", :params => {"-" => nil, nil => "postgres", "-c" => "#{PostgresAdmin.scl_enable_prefix} \"#{cmd}\""})
+      conn = PG.connect(:user => "postgres", :dbname => "postgres")
+      conn.exec("CREATE DATABASE #{database} OWNER #{username} ENCODING 'utf8'")
```

Comment:
@carbonin thanks. I do remember adding that line.

I guess if we need to change the privilege of the user who is running `appliance_console`, we can cross that bridges another day.

good :scissors: - much easier to read